org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/preferences/ClientPreferences.java

```diff
@@ -209,6 +209,10 @@ public boolean isAdvancedExtractRefactoringSupported() {
 		return Boolean.parseBoolean(extendedClientCapabilities.getOrDefault("advancedExtractRefactoringSupport", "false").toString());
 	}
 
+	public boolean isAdvancedIntroduceParameterRefactoringSupported() {
```

Comment:
Why is this needed? there's no need for client to implement a UI mechanism behind, all should be able to use this feature without extra config

Reply:
We use this property to trigger the parameter rename, after the code action has been applied.

Reply:
sorry I was actually testing the wrong thing. You're right, it works as expected